src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -120,7 +122,8 @@ function WorkspaceSettingsPage(props) {
                             type={CONST.ICON_TYPE_WORKSPACE}
                             fallbackIcon={Expensicons.FallbackWorkspaceAvatar}
                             style={[styles.mb3]}
-                            anchorPosition={{top: 172, right: 18}}
+                            anchorPosition={{horizontal: props.windowWidth - 355, vertical: 250}}
+                            anchorAlignment={{horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.CENTER, vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.TOP}}
```

Comment:
Something is not right here (same goes with ProfilePage)
![Screenshot from 2023-05-17 19-25-30](https://github.com/Expensify/App/assets/16493223/56cb3804-2a5f-4bf8-8cda-efb95ec5e26b)